index.md

```diff
@@ -104,9 +104,14 @@ allocation. GlareDB Cloud is available to everyone with a
 
 See our [Pricing page] for more details on available plans.
 
+[S3]: /docs/data-sources/supported/s3/
```

Comment:
do these links get rendered in the source using the _actualy_ domain and path or do we rely on redirection (and does that matter for SEO?)

Reply:
These get rendered in the source. And we have a sitemap so seo should be fine (https://docs.glaredb.com/sitemap.xml)